docs/triply-api/index.md

```diff
@@ -824,8 +824,406 @@ Result:
+
+
+#### Global Object identification
+For reasons such as caching, the user should be able to query an object by their unique ID. This is possible using global object identification, using the `node(id:ID)` query. 
+An example:
+
+```graphql
+{
+  node(id: "https://example.org/book/Odyssey") {
+    id
+  }
+}
+```
+
+For more information on global object identification, see [graphql specification](https://graphql.org/learn/global-object-identification/). 
+
+#### Pagination
+A simple query would be:
+ ```graphql
+{
+  BookConnection {
+    edges {
+      node {
+        id
+        title
+      }
+    }
+  }
+}
+ ```
+The results would include the IRIs of books together with their titles and would be paginated.
+
+In order to paginate through a large number of results, our GraphQL implementation supports **cursor-based pagination using connections**. For more information, please visit the Relay project's [cursor-based connection pagination specification](https://relay.dev/graphql/connections.htm).
+
+#### Filtering
+When you query for objects, you might want to get back objects based on specific values in certain fields. You can do this by filtering.
+#### Simple cases
+For example, you can query for people with a specific id:
+```graphql
+{
+  PersonConnection(filter: {id: "https://example.org/person/Homer"}) {
+    edges {
+      node {
+        id
+        name
+      }
+    }
+  }
+}
+```
+Another query would be to search for a person with a specific name:
+```graphql
+{
+  PersonConnection(filter: {name: {eq: "Homer"}}) {
+    edges {
+      node {
+        id
+        name
+      }
+    }
+  }
+}
+```
+
+Notice that in the second example, there is a new field for filtering called `eq`. When we want to filter on a field with returns a scalar, meaning that its value is represented by a literal in linked data, we have to use comparison operators: `eq`,`in` for equality, and `notEq` and `notIn` for inequality. The operators `in` and`notIn` are refering to lists.
+
+On the other hand, when we are filtering based on IDs - or in linked data terms, based on the IRI - , as in the first example, we don't use comparison operators.
+
+
+The only idiomatic case is the literal with a language tag and `rdf:langString` as a datatype. This literal is represented as ` { value: "example-string", language: "en" }` and the corresponding scalar is `RdfsLangString` . This means that in order to filter using a value of this scalar type, you have to execute the query below:
+
+```graphql
+{
+  PersonConnection(filter: {name: {eq: {value: "Ulysses", language: "en"}}}) {
+    edges {
+      node {
+        id
+        name
+      }
+    }
+  }
+}
+```
+##### Language filtering
+Additionally, there is support for filtering results based on the language tag.
+An example is:
+- Linked data:
+```turtle
+person:Ulysses a sdo:Person;
+              sdo:name "Ulysses"@"en";
+              sdo:name "Oδυσσέυς"@"gr".
```

Comment:
I think the English name is [Odysseus](https://en.wikipedia.org/wiki/Odysseus) (Nice example :D)

Reply:
https://en.wikipedia.org/wiki/Ulysses
I didn't make this up!

Reply:
But I can change it !